.github/workflows/ci.yml

```diff
@@ -88,7 +88,7 @@ jobs:
       - name: Archive app
         run: >
           pushd ~/go/src/github.com/digitalbitbox/bitbox-wallet-app/frontends/qt/build/osx;
-          zip -r ${{github.workspace}}/BitBoxApp-macos.zip BitBox.app;
+          zip -y -r ${{github.workspace}}/BitBoxApp-macos.zip BitBox.app;
```

Comment:
Actually we should use `ditto`

https://github.com/digitalbitbox/bitbox-wallet-app/pull/903#discussion_r448808691

as `zip` seems to modify the app bundle in undesired ways. `ditto` behaves like the Finder when compressing.

Reply:
Done.